deposits.js

```diff
@@ -0,0 +1,6 @@
+  function getDetails(){
+    let dname = document.getElementById("name");
+    let damount=document.getElementById("amount");
+    fetch("/deposit", {method: "POST", {name: dname, amount: damount}}).then(()=>{})
```

Comment:
What do we do inside the then function?

Reply:
take the name , the amount  and send it to the back.

Reply:
I see and empty function body in the `then`

Reply:
updating that in a few